shell/src/main/java/alluxio/shell/command/FileInfoCommand.java

```diff
@@ -58,12 +57,34 @@ protected void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioExcepti
   }
 
   @Override
+  protected void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioException, IOException {
+    URIStatus status = mFileSystem.getStatus(path);
+    if (status.isFolder()) {
+      List<URIStatus> childrens = mFileSystem.listStatus(path);
```

Comment:
I know ./bin/alluxio fs ls /path/to/dir/ can list all children of a directory, but the detail info can not displays. So, I use a new options for user to decide whether need to show all the directory's children, and default just displays directory's info.